packages/example-getting-started/docs/putting-it-together.md

```diff
@@ -0,0 +1,89 @@
+### Putting it all together
+
+We've got all of our artifacts now, and all that's left is to bind them to our
+[Application](http://loopback.io/doc/en/lb4/Application.html) so that LoopBack's
+[Dependency injection](http://loopback.io/doc/en/lb4/Dependency-injection.html)
+system can tie it all together for us!
+
+LoopBack's
+[boot module](https://github.com/strongloop/loopback-next/tree/master/packages/boot)
+will automatically discover our controllers, repositories, datasources and
+other artifacts and inject them into our application for use.
+
+>**NOTE**: The boot module will only discover and inject artifacts that
+follow our established conventions for artifact directories. Here are some
+examples:
+>* Controllers: `./src/controllers`
+>* Datasources: `./src/datasources`
+>* Models: `./src/models`
+>* Repositories: `./src/repositories`
+
+#### src/application.ts
+```ts
+import {ApplicationConfig} from '@loopback/core';
+import {RestApplication} from '@loopback/rest';
+import {db} from './datasources/db.datasource';
+
+/* tslint:disable:no-unused-variable */
+// Do not remove!
+// Class and Repository imports required to infer types in consuming code!
+// Binding and Booter imports are required to infer types for BootMixin!
+import {BootMixin, Booter, Binding} from '@loopback/boot';
+import {
+  Class,
+  Repository,
+  RepositoryMixin,
+} from '@loopback/repository';
+/* tslint:enable:no-unused-variable */
+
+export class TodoApplication extends BootMixin(
+  RepositoryMixin(RestApplication),
+) {
+  constructor(options?: ApplicationConfig) {
+    super(options);
+    this.projectRoot = __dirname;
```

Comment:
New projects also list the conventions here to show users the defaults and what they can change.

https://github.com/strongloop/loopback-next/blob/master/packages/cli/generators/app/templates/src/application.ts.ejs#L17-L26